src/lib/utils/token-speed.svelte.ts

```
@@ -0,0 +1,43 @@
+import { untrack, type Component } from 'svelte';
+
+interface TokenSpeedControls {}
+
+interface TokenSpeedProps {
+	content: string;
+	sep: string;
+}
+
+export interface TokenSpeed extends TokenSpeedProps {}
+export class TokenSpeed {
```

Biome (2.1.2)

[error] This class is unsafely merged with an interface.

The interface is declared here.

The TypeScript compiler doesn't check whether properties defined in the interface are initialized in the class.

(lint/suspicious/noUnsafeDeclarationMerging)